README.md

```diff
@@ -1,5 +1,48 @@
 # New Relic Infrastructure Integrations
 
+**DEPRECATION NOTICE AND UPGRADE INSTRUCTIONS**
+
+This package was born at a time when the New Relic Infrastructure team was just
```

Comment:
`[suggestion]`

This package was created at the time when New Relic Infrastructure was starting with a reduced number of developers. At that moment it made sense having a mono repository with all our integrations in order to install them all in once. However, things changed during the time and it ended up with a completely different scenario having a larger amount of integrations. 

For this reason, we took the approach by having a different repository for each integration which allows us more control, faster delivery and it's easier to maintain and it causes less errors.